robber/matchers/truthy.py

```diff
@@ -10,15 +11,9 @@ class Truthy(Base):
     def matches(self):
         return bool(self.actual)
 
-    def failure_message(self):
-        message = 'Expected "{actual}" to be {adj}'
-
-        if not self.is_negative:
-            adj = 'truthy'
-        else:
-            adj = 'falsy'
-
-        return message.format(actual=self.actual, adj=adj)
+    @property
+    def explanation(self):
+        return Explanation(self.actual, self.is_negative, action="be truthy", negative_action='be falsy')
```

Comment:
Whatever you are using, stick to 1, `'` and `"` are both good, but it needs to be consistent.